lib/jormungandr/test/integration/Cardano/Wallet/Jormungandr/NetworkSpec.hs

```diff
@@ -149,6 +168,58 @@ spec = do
                     shouldThrow io $ \(ErrUnexpectedNetworkFailure link _) ->
                         show link == show (safeLink api (Proxy @GetTipId))
             bracket (startNode wrongUrl wait) killNode test
+
+
+    -- NOTE: 'Right ()' just means that the format wasn't obviously wrong.
+    -- The tx may still be rejected.
+    describe "Submitting signed transactions" $ beforeAll startNode' $ afterAll killNode $ do
+        it "empty tx succeeds" $ \(_, nw) -> do
+            let signedEmpty = (Tx [] [], [])
+            runExceptT (postTx nw signedEmpty) `shouldReturn` Right ()
+
+        it "tx from faucet succeeds" $ \(_, nw) -> do
+            let (Right mw) = mkMnemonic @15 ["tattoo","potato","foil","mutual","slab","path","forward","pencil","suit","marble","hill","meat","close","garden","bird"]
```

Comment:
Let go for `generateEntropy` here instead of a hard-coded mnemonic :+1:

Reply:
Okay, just saw the title now as I was wondering why you'd go for hard-coded addresses in the next tests but not in this one.
We shouldn't hard-coded a generated faucet mnemonic here, there are located in an opaque module for a reason :), they might (and will actually) change over time and they don't have any special semantic so no need to consider them in any particular way.

I think the test would read better if you simply provide some arbitrary inputs and outputs and show that a well-formed transaction can be submitted.